staging/src/k8s.io/apiserver/pkg/admission/plugin/gc/gc_admission_test.go

```diff
@@ -204,140 +201,140 @@ func TestGCAdmission(t *testing.T) {
 		{
 			name:       "super-user, create, no objectref change",
 			username:   "super",
-			resource:   api.SchemeGroupVersion.WithResource("pods"),
-			newObj:     &api.Pod{},
+			resource:   corev1.SchemeGroupVersion.WithResource("pods"),
+			newObj:     &corev1.Pod{},
```

Comment:
technically an admission plugin will never see this versioned type, but instead some internal type (like the one that was imported here previously). But it does not really matter for the plugin as it is generic.